packages/react-devtools-shared/src/devtools/views/ErrorBoundary/ErrorBoundary.js

```diff
@@ -49,16 +49,16 @@ export default class ErrorBoundary extends Component<Props, State> {
     const errorMessage =
       typeof error === 'object' &&
       error !== null &&
-      error.hasOwnProperty('message')
+      typeof error.message === 'string'
         ? error.message
-        : String(error);
+        : null;
```

Comment:
This change prevents us from searching for a matching GitHub issue when we have no unique error message to search with.